### PR TITLE
Translations for moods in ja.yml

### PR DESCRIPTION
**Pull request by matcha440**
# Description 
This one is very short, but I found them somewhat difficult.  Hopefully through the review we can improve these.

## Corresponding Issue
#1215 

# Test Coverage
🚫

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -287,24 +287,24 @@ ja:
...
-      premade4_description: 'Not accepting certain facts, feeling angry, vengeful, and/or jealous'
-      premade5_name: Hopeless
-      premade5_description: 'Feeling sad and hopeless - unable to find any possible solutions'
-      instructions: 'You haven''t created any custom moods yet.<br>You can also add the following premade moods and customize them another time.'
+      subtitle: 'あなたの心の状態、気分、感情を記述するタグを作成します。'
+      premade1_name: 承諾
+      premade1_description: '事実には平和にいます。'
+      premade2_name: 野心的
+      premade2_description: '目標には希望があります。'
+      premade3_name: 好奇心
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      premade3_name: 気になっている
```

好奇心 is a big and formal word that wouldn't really be used outside a dictionary imo, so maybe a more casual approach would be better here.

File: config/locales/ja.yml [hunk 2]
```diff
@@ -287,24 +287,24 @@ ja:
...
+      premade3_description: '不確実性を知りたいです。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      premade3_description: '何何が知りたい'
```

In the same casual spirit.

File: config/locales/ja.yml [hunk 3]
```diff
@@ -287,24 +287,24 @@ ja:
...
+      premade4_name: 恨みがましい
+      premade4_description: '事実を否定します。怒り、恨み、嫉妬を感じます。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      premade4_description: '事実を否定している。怒り、恨み、嫉妬などを感じている。'
```

File: config/locales/ja.yml [hunk 4]
```diff
@@ -287,24 +287,24 @@ ja:
...
+      premade5_name: 切望的
+      premade5_description: '悲しい感じです。何もできないと考えます。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      premade5_description: '悲しい。何もできないと考えざるが得ない。'
```

File: config/locales/ja.yml [hunk 5]
```diff
@@ -287,24 +287,24 @@ ja:
...
+      instructions: '気分はまだ作成しませんでした。<br>後でカスタマイズしたら、この作り置きの気分を使えます。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      instructions: '気分はまだ作成してません。<br>後でカスタマイズしたら、この作り置きの気分を使えます。'
```

**Comment by matcha440** [comment]
Very helpful, as always.  Thank you!

**Pull request merged**